files/conf/containerd-config.toml

```diff
@@ -40,3 +40,9 @@ sandbox_image = "{{ .Images.Pause }}"
 [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
 endpoint = ["{{index .RegistryMirrors 0}}"]
 {{- end }}
+
+[plugins."io.containerd.grpc.v1.cri".registry.configs."docker.io".auth]
+auth = "{{ .DockerhubToken }}"
```

Comment:
Is the `DockerhubToken` variable always populated or do we maybe need to check that?

Reply:
The token is populated here https://github.com/giantswarm/k8scloudconfig/blob/master/files/conf/kubelet-docker-config.json#L1-L7

Reply:
AFAIK we don't check it there, so I assume it is safe